src/RoomsSDKAdapter.integration.test.js

```diff
@@ -1,42 +1,45 @@
 import {createIntegrationTestUser, removeIntegrationTestUser} from './testHelper';
-import RoomsSDKAdapter from './RoomsSDKAdapter';
+
+import WebexSDKAdapter from './';
 
 describe('Rooms SDK Adapter', () => {
-  let roomsSDKAdapter, subscription, user;
+  let createdRoom, getRoom$, webexSDKAdapter, subscription, user;
 
   // Since these are integration tests with live data,
   // increase the async "idle" timeout so jest doesn't error early.
   jest.setTimeout(30000);
 
   beforeAll(async () => {
     user = await createIntegrationTestUser();
-  });
-
-  beforeEach(() => {
-    roomsSDKAdapter = new RoomsSDKAdapter(user.sdk);
+    webexSDKAdapter = new WebexSDKAdapter(user.sdk);
+    await webexSDKAdapter.connect();
   });
 
   afterAll(async () => {
-    // This is done after unsubscribing, but since unsubscribe can't be async, manually call it.
-    await user.sdk.rooms.stopListening();
```

Comment:
Can you run `npm run test:integration` to verify the tests exit gracefully with this removed? I had to add this in order to get jest to exit properly. Just running `npm test` isn't enough to verify.

Reply:
i did.